crates/ubrn_cli/src/codegen/mod.rs

```diff
@@ -264,7 +264,7 @@ mod files {
     templated_file!(IndexTs, "index.ts");
```

Comment:
Should this also end in `.tsx`?

Reply:
Yes, you're right. I renamed the template file and made this `index.tsx` to match. I also renamed the `IndexTs` struct.

Thanks for prompting me out of laziness. :)